controllers/cluster_controller.go

```diff
@@ -218,3 +227,12 @@ func (r *ClusterReconciler) ensureClusterOwnerReferences(ctx context.Context, cl
 
 	return nil
 }
+
+func (r *ClusterReconciler) SetBundlesRef(ctx context.Context, clus *anywherev1.Cluster) error {
```

Comment:
does this need to be public?

Reply:
I made this public for unit testing

Reply:
let's test from reconcile and make this private